omlmd/cli.py

```diff
@@ -26,9 +29,15 @@ def cli():
 @cli.command()
 @plain_http
 @click.argument("target", required=True)
-@click.option("-o", "--output", default=".", show_default=True)
+@click.option(
+    "-o",
+    "--output",
+    default=Path.cwd(),
+    show_default=True,
+    type=click.Path(path_type=Path, exists=True, resolve_path=True),
```

Comment:
I believe here lies an issue as highlighted by E2E, the output path was previously created if unexisting